llm4pddl/approaches/llm_open_loop_approach.py

```diff
@@ -122,6 +135,12 @@ def _create_prompt(task: Task, plan: Optional[Plan] = None) -> str:
             # Create the goal string.
             goal_str_groups = utils.group_by_predicate(problem.goal)
             goal_str = "\n".join(sorted(goal_str_groups))
+        # Randomize objects in init, goal, solution strings.
```

Comment:
would there be any downside to calling the utility once on the `prompt` below instead?